source/sample.apib

```diff
@@ -0,0 +1,4 @@
+# GET /message
```

Comment:
Let's make this the following:

```apib
## /message
### GET
```

That way, we can reuse `sample.apib` in `/compose` part of the blueprint too.